plugins/devtools/app-config.dynamic.yaml

```diff
@@ -0,0 +1,19 @@
+dynamicPlugins:
+  frontend:
+    backstage.plugin-devtools:
+      # For the sidebar link
+      dynamicRoutes:
+        - importName: DevToolsPage
+          menuItem:
+            text: DevTools
+            icon: chat
+          path: /devtools
+      # For showing in the overview tab
+      mountPoints:
```

Comment:
Is it related to a catalog entity ? Do we really want to put that in the catalog entity tabs ?

Reply:
I thought they're using it in a tab (Overview in RHDH, but it was a custom tab in their Backstage build). I could be mistaken so I'm clarifying with them atm.

Reply:
Just confirmed with them and they're using it from the sidebar only. I'll remove the one from `mountPoints`.